networks/suzuka/suzuka-client/log_file.txt

```diff
@@ -0,0 +1,15 @@
+[2m2024-09-12T09:29:14.765310Z[0m [33m WARN[0m [2msuzuka_client::load_soak_testing[0m[2m:[0m Error during scenario spawning: task 14 panicked with message "Lazy instance has previously been poisoned"
```

Comment:
The file is still here. Please modify your git procedures to not commit just everything that happens to be in the dirty working tree.